src/yggdrasil/session.go

```diff
@@ -187,9 +189,9 @@ func (ss *sessions) createSession(theirPermKey *crypto.BoxPubKey) *sessionInfo {
 	sinfo.mySesPriv = *priv
 	sinfo.myNonce = *crypto.NewBoxNonce()
 	sinfo.theirMTU = 1280
-	ss.router.core.config.Mutex.RLock()
-	sinfo.myMTU = uint16(ss.router.core.config.Current.IfMTU)
-	ss.router.core.config.Mutex.RUnlock()
+	// TODO: sinfo.myMTU becomes unnecessary if we always have a reference to the
```

Comment:
I think `sinfo.myMTU` is still necessary for thread safety, unless we can guarantee that `ss.myMaximumMTU` is constant.